ci/lint.yml

```
@@ -0,0 +1,15 @@
+---
+
+platform: linux
+
+image_resource:
+  type: docker-image
+  source:
+    repository: golang
+    tag: latest
+
+inputs:
+  - name: dp-download-service
+
```

Consider adding a cache declaration so that concourse doesn't have to download the go dependencies every time.
```suggestion
caches:
  - path: go/
```